go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/edgexfoundry/device-uart
+
+require (
+	github.com/edgexfoundry/device-sdk-go/v2 v2.0.0
```

Comment:
```suggestion
	github.com/edgexfoundry/device-sdk-go/v2 v2.1.0
```

Reply:
I suggest upgrading to `github.com/edgexfoundry/device-sdk-go/v2 v2.2.0-dev.13` directly